src/OrchardCore.Modules/OrchardCore.OpenId/ViewModels/CreateOpenIdApplicationViewModel.cs

```diff
@@ -26,8 +26,8 @@ public class CreateOpenIdApplicationViewModel : IValidatableObject
 
         public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
         {
-            return OpenIdUrlValidator.ValidateUrls(nameof(RedirectUris), RedirectUris)
-                .Union(OpenIdUrlValidator.ValidateUrls(nameof(PostLogoutRedirectUris), PostLogoutRedirectUris));
+            return validationContext.ValidateUrls<CreateOpenIdApplicationViewModel>(nameof(RedirectUris), RedirectUris)
```

Comment:
You could use inline lambda expression

Reply:
the PR was to add translation in validation messages from OpenId. I don't see any added value changing to inline lamda expression here.

Reply:
> the PR was to add translation in validation messages from OpenId

Yep, but there 're many formatting fixes that you did ;)

> I don't see any added value changing to inline lamda expression here.

We usually used lambda when the function contains one line

Reply:
although it is more readable as a function :)

Reply:
The important here is the localization stuff ;)